covalent/_workflow/transport.py

```diff
@@ -312,6 +312,10 @@ def __init__(self) -> None:
             "stderr": None,
         }
 
+    def add_node_by_id(self, node_id: int, **attr):
```

Comment:
Is this used anywhere?

Reply:
@cjao I believe it was... I'll double check